sphinx-docs/usermanual/Documentation for Documentation lol

```diff
@@ -0,0 +1,22 @@
+What I Changed:
```

Comment:
Was this supposed to be a commit message? You can use `git commit -F <filename>`. Otherwise you might consider moving and renaming this file... something like `sphinx-docs/README` or `sphinx-docs/TODO`.